setup.py

```diff
@@ -21,6 +21,7 @@
     install_requires=[
         'attrs',
         'requests',
+        'requests_mock',
```

Comment:
Might want to change the url in this file too :)

Reply:
Oh, that's right!

I'll do it, thanks!

Reply:
[Done](https://github.com/release-engineering/starmap-client/compare/b26cde7d34c098cbb3db6cac36b0e717e6655499..f3c19356e31b28af00b8fd2e280ec49d552eb3ae) 😄